unified-codes/apps/web/.storybook/config.js

```diff
@@ -1,5 +1,18 @@
-import { configure, addDecorator } from '@storybook/react';
+import { addDecorator, addParameters, configure } from '@storybook/react';
 import { withKnobs } from '@storybook/addon-knobs';
 
+const webContext = require.context('..', true, /.stories\.(j|t)sx?$/);
+const libContext = require.context('../../../libs/ui/src/lib', true, /.stories\.(j|t)sx?$/);
+const parameters = {
```

Comment:
Looks and sounds so promising 😿. Is it better to remove the config in this case then? (or a comment to forewarn unsuspecting developers who may think they are going crazy? 😁). Only speaking as someone who often questions their own sanity when reading code.

Reply:
Hm on the topic of questioning sanity, I've been testing more and I can actually see them sorted alphabetically when I move the addParameters line before the configure line? I don't know why yet.

Reply:
I have seen it work now! - so will leave it in for now, and try to figure out why it works sometimes

Reply:
Ah, that makes sense, so it wasn't just my imagination. Phew